content/posts/2025-07-17-whats-new-1.9/index.md

```diff
@@ -0,0 +1,197 @@
+tags: network,observability,IPsec,filter,query,udn,cli
+authors: [stleerh]
+---
+
+[Network Observability 1.9](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/network_observability/index) is an optional operator that provides insights into your network traffic, including features like packet drops, latencies, DNS tracking, and more.  You can view this in the form of graphs, a table, or topology.
+
+This version aligns with [Red Hat OpenShift Container Platform (OCP) 4.19](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19).  If you've used OCP before, you'll notice the new look-and-feel.  Network Observability blends in nicely with this new UI.
+
+While Network Observability 1.9 is backwards-compatible with older OCP and Kubernetes releases, it is recommended that you use OCP 4.19.  For installation instructions, check out the documentation on [OpenShift Container Platform](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19#Install) and [Network Observability](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/network_observability/installing-network-observability-operators).
+
+This article covers the new features in this release, namely IPsec tracking, flowlogs-pipeline filter query, UDN Mapping, and Network Observability CLI enhancements.  If you want to learn about the past features, read my older [What's new in Network Observability](https://developers.redhat.com/author/steven-lee) articles.
+
+## IPsec tracking
+
+Network Observability can identify IPsec traffic flows and indicate if they were successfully encrypted or not.  To try this out, you need to have OVN-Kubernetes as your Container Network Interface (CNI), which is the default for OCP.
+
+To enable IPsec, follow the instructions on [Configuring IPsec encryption](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/networking/network-security#configuring-ipsec-ovn).  If you didn't enable IPsec during cluster installation, it's a bit tricky to set up, so I've provided a quick guide below on setting this up in a test environment.
+
+Network Observability can identify encrypted IPsec traffic between pods.  In the OpenShift web console, when you create the FlowCollector instance, scroll down to the **Agent configuration** and open up the section named **Features**.  In the dropdown for **Value**, select  **IPSec** as shown in Figure 1.
+
+![Enable IPsec eBPF feature](ebpf_features-ipsec.png)<br>
+Figure 1: Enable IPsec eBPF feature
+
+The equivalent in YAML is to add this in the **spec** section:
+
+```yaml
+spec:
+  agent:
+    ebpf:
+      features:
+      - IPSec
+```
+
+### Network Observability - IPsec feature
+
+In **Observe > Network Traffic, Traffic flows tab**, it adds a new column **IPSec Status** that has the possible values of "success", "error", or "n/a" (Figure 2).
```

Comment:
btw, on "IPsec" versus "IPSec": I know the former is correct for the acronym. I guess it's more debatable when it's part os a json/camel-case name, such as `IPSecStatus`;
I've opened this PR to fix just the column/filter display name, without changing the underlying json name: https://github.com/netobserv/network-observability-operator/pull/1778